ports/espressif/boards/m5stack_atom_u/sdkconfig

```diff
@@ -0,0 +1,27 @@
+CONFIG_ESP32_ECO3_CACHE_LOCK_FIX=y
```

Comment:
This is now in `esp-idf-config/sdkconfig-esp32.default`, so it doesn't need to be in individual board `sdkconfig`s.
```suggestion
```

Reply:
@dhalbert I saw that you requested the code change above and you made yourself the change in commit 01f6762aa1110bf489f39ec0ed2589a277dd78ca but github still show that the merge is blocked with this message: "Merging can be performed automatically once the requested changes are addressed."
Can you please help me to understand what shall I do?